script/setup

```diff
@@ -48,11 +46,6 @@ if [ ! -f config/application.yml ]; then
   printf "${YELLOW}Copied config/application.yml Make sure to fill it with the appropriate configuration values.\n\n${NO_COLOR}"
 fi
 
-# Create the development database user
-if ! psql -c '\du' -t | grep $DB_USER > /dev/null; then
-  psql -c "CREATE USER ${DB_USER} WITH SUPERUSER PASSWORD '${DB_PASSWORD}';"
```

Comment:
I can see why removing the 9.5 version constraint might help some people, but what's the point of removing this from the script and adding an extra manual step for setup?

Reply:
Especially given that people can easy miss-type the password. ^

Reply:
This step required the current user to execute admin commands with psql. My dev user may not be able to do that. I would switch with `su` and do it manually. But since this is a dev environment and the password is not secure anyway, I would provide the user with the full command including the password so that they don't have to type it.

Reply:
that would be ideal but I didn't find the way to provide the password within the command using `createuser`. Feel free to suggest a better way.

Reply:
In that case I would go with `psql -c "CREATE USER...`.